src/timepicker/timepicker.js

```diff
@@ -242,6 +242,7 @@ angular.module('ui.bootstrap.timepicker', [])
     controller:'TimepickerController',
     replace: true,
     scope: {},
+    priority: 1,
```

Comment:
Setting the priority has a huge code smell, and likely is not the correct solution.  This would be a breaking change.

Even if setting the priority is the approach to go, setting it to 1 is no good - we would need to choose a more appropriate value.